packages/plugins/typescript-aws-amplify-angular/src/index.ts

```diff
@@ -0,0 +1,46 @@
+import { Types, PluginValidateFn, PluginFunction } from '@graphql-codegen/plugin-helpers';
+import { visit, GraphQLSchema, concatAST, Kind, FragmentDefinitionNode, OperationDefinitionNode } from 'graphql';
+import { RawClientSideBasePluginConfig } from '@graphql-codegen/visitor-plugin-common';
+import { AwsAmplifyAngularVisitor } from './visitor';
+import { extname } from 'path';
+
+export interface AwsAmplifyAngularRawPluginConfig extends RawClientSideBasePluginConfig {}
```

Comment:
If there is no additional configuration, you can just use `RawClientSideBasePluginConfig` directly.